app/component/MobileTicketPurchaseInformation.js

```diff
@@ -21,7 +21,7 @@ export default function MobileTicketPurchaseInformation(
     !fare.isUnknown,
     config.availableTickets,
   );
-  const price = `${(fare.cents / 100).toFixed(2)} €`.replace('.', ',');
+  const price = `${fare.price.toFixed(2)} €`.replace('.', ',');
```

Comment:
Do we need toFixed?

Reply:
It seems that without it, the UI will display e.g. 2.7 € instead of the 2.70 € that has been previously shown. OTP returns the price with only one digit.